src/frontend/static/bulma/css/style.css

```diff
@@ -374,7 +377,7 @@ a:hover {
 
 code {
   background-color: whitesmoke;
-  color: #f14668;
+  color: #da1039;
```

Comment:
Out of curiosity, what drived this change / upgrade of bulma ?

Reply:
I wanted to use some helper css classes in bulma ("m-*") but I discovered they weren't there yet in our version.